### ISSUE TITLE
0.7.58: Query returns empty result on overpass-api.de only

### ISSUE THREAD

**Issue by mmd-osm** [drolbr/Overpass-API#667]
The following query returns an empty result on overpass-api.de (which runs on 0.7.58 atm). Expected result would be around 1900 relations (confirmed on other public instances, such as kumi).

```
[bbox:50.819818262156545,6.330871582031249,51.70320428462806,7.2564697265625];
rel[type=multipolygon](if:count_by_role("outer") == count_members(ways) && count_by_role("outer") > 0 && count_members() == count_members(ways));
out tags;
```

**Comment by drolbr** [comment]
Fixed in 2b5354b14bd520d6cdefea30d2f70c9be4b9cf23

Thank you for pointing me to this problem.

**Comment by mmd-osm** [comment]
Thank you for the quick fix. I'm happy to confirm that the issue no longer occurs.
